packages/components/src/range-control/styles/range-control-styles.js

```diff
@@ -0,0 +1,310 @@
+ */
+import { css } from '@emotion/core';
+import styled from '@emotion/styled';
+
+/**
+ * Internal dependencies
+ */
+import { color } from '../../utils/colors';
+
+const rangeHeight = () => css( { height: 30, minHeight: 30 } );
+const rootRtl = ( { isRTL } ) => css( { direction: isRTL ? 'rtl' : 'ltr' } );
+
+export const Root = styled.span`
+	-webkit-tap-highlight-color: transparent;
+	box-sizing: border-box;
+	cursor: pointer;
+	align-items: flex-start;
+	display: inline-flex;
+	justify-content: flex-start;
+	padding: 0;
+	position: relative;
+	touch-action: none;
+	width: 100%;
+
+	${rootRtl}
+`;
+
+const wrapperMargin = ( { marks } ) =>
+	css( { marginBottom: marks ? 16 : null } );
+
+export const Wrapper = styled.span`
+	box-sizing: border-box;
+	color: ${color( 'blue.medium.focus' )};
+	display: block;
+	padding-top: 15px;
+	position: relative;
+	margin-left: 10px;
+	width: 100%;
+
+	${rangeHeight};
+	${wrapperMargin};
+
+	[dir='rtl'] & {
+		margin-right: 10px;
+	}
+`;
+
+export const BeforeIconWrapper = styled.span`
+	margin-right: 6px;
+	margin-top: 5px;
+
+	> .dashicon {
+		margin: 0;
+	}
+`;
+
+export const AfterIconWrapper = styled.span`
+	margin-left: 16px;
+	margin-top: 5px;
+
+	> .dashicon {
+		margin: 0;
+	}
+`;
+
+export const Rail = styled.span`
+	background-color: ${color( 'lightGray.600' )};
+	box-sizing: border-box;
+	left: 0;
+	pointer-events: none;
+	right: 0;
+	display: block;
+	height: 3px;
+	position: absolute;
+	margin-top: 14px;
+	top: 0;
+`;
+
+export const Track = styled.span`
+	background-color: currentColor;
+	border-radius: 1px;
+	box-sizing: border-box;
+	height: 3px;
+	pointer-events: none;
+	display: block;
+	position: absolute;
+	margin-top: 14px;
+	top: 0;
+`;
+
+export const MarksWrapper = styled.span`
+	box-sizing: border-box;
+	display: block;
+	position: relative;
+	width: 100%;
+	user-select: none;
+`;
+
+const markFill = ( { isFilled } ) => {
+	return css( {
+		backgroundColor: isFilled ? 'currentColor' : color( 'lightGray.600' ),
+	} );
+};
+
+export const Mark = styled.span`
+	box-sizing: border-box;
+	height: 9px;
+	left: 0;
+	position: absolute;
+	top: -4px;
+	width: 1px;
+
+	${markFill};
+`;
+
+const markLabelFill = ( { isFilled } ) => {
+	return css( {
+		color: isFilled ? color( 'darkGray.300' ) : color( 'lightGray.600' ),
+	} );
+};
+
+export const MarkLabel = styled.span`
+	box-sizing: border-box;
+	color: ${color( 'lightGray.600' )};
+	left: 0;
+	font-size: 11px;
+	position: absolute;
+	top: 12px;
+	transform: translateX( -50% );
+	white-space: nowrap;
+
+	${markLabelFill};
+`;
+
+export const ThumbWrapper = styled.span`
+	align-items: center;
+	margin-left: -10px;
+	margin-top: 5px;
+	width: 20px;
+	box-sizing: border-box;
+	display: flex;
+	height: 20px;
+	justify-content: center;
+	outline: 0;
+	position: absolute;
+	pointer-events: none;
+	top: 0;
+	user-select: none;
+
+	[dir='rtl'] & {
+		margin-left: 0;
+		margin-right: -10px;
+	}
+`;
+
+const handleReducedMotion = () => {
```

Comment:
Feels like something that could be moved to the utils folder (like color) and should we align its implementation with the SASS version we have?

Reply:
Ah, good idea 👍